less/home-page.less

```diff
@@ -58,12 +87,28 @@
       }
 
       .btn {
+        .button(@btn-bg-color, @porcelain);
+
+        
+
+        @media @phone {
+          .button(@btn-bg-color, @porcelain);
```

Comment:
since you already are styling the button, no need to restyle it again in your media query.